Kickstarter-iOS/Views/Controllers/PledgeContinueViewController.swift

```diff
@@ -25,10 +25,7 @@ final class PledgeContinueViewController: UIViewController {
       |> checkoutBackgroundStyle
 
     _ = self.continueButton
-      |> greenButtonStyle
-      |> UIButton.lens.title(for: .normal) %~ { _ in
-        Strings.Continue()
-      }
+      |> continueButtonStyle
```

Comment:
Moving to `// MARK: - Styles` for consistency

Reply:
I was thinking that we can start renaming the `Styles` pragma mark above `bindStyles()` to `Styles Binding` or `Bind Styles`, so it differs from the `Styles` on the bottom of the file. what do you think?